Criterion/Report.hs

```diff
@@ -128,11 +123,8 @@ formatReport reports templateName = do
     let context = object
             [ "json"                .= reports
             , "report"              .= reports'
```

Comment:
I don't think the new templates make use of `report`, right? If so, can we remove this line?

Reply:
Fixed in d381808